openedx_caliper_tracking/utils.py

```diff
@@ -127,23 +126,24 @@ def send_notification(key, data, subject, from_email, dest_emails):
     TEMPLATE_PATH = 'openedx_caliper_tracking/{key}_email.html'
     content = json.dumps(data)
     email_template_path = TEMPLATE_PATH.format(key=key)
-    html_content = get_template(email_template_path).render(data)
+    from edxmako.shortcuts import render_to_response
```

Comment:
Move this with other imports.

Reply:
@tehreem-sadat 
Here it is the external dependency. To get rid of the `module not found` error while testing the app independently I have to keep this import statement inside the function body.

Moreover this function has been modified in this PR. Kindly have a look at it
[https://github.com/ucsd-ets/openedx-caliper-tracking/pull/35](https://github.com/ucsd-ets/openedx-caliper-tracking/pull/35)